.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
```

Comment:
```suggestion
[*]
charset = utf-8
end_of_line = lf
trim_trailing_whitespace = true
max_line_length = 120
# visual setting for intelliJ IDE, no consequences on code validation
# https://www.jetbrains.com/help/idea/editorconfig.html
ij_visual_guides = 100,120,150
```

Reply:
Fixed

Reply:
Did I miss something or you didn't push yet?

Reply:
I didn't post it - I was waiting for the rest of the comments to be closed so I could post it in one commit